apps/migrator-e2e/src/integration/migrator.spec.ts

```diff
@@ -11,6 +11,7 @@ const clearProtractor = (): Cypress.Chainable<JQuery<HTMLElement>> => {
 
 const enterProtractor = (value: string): void => {
   cy.get('textArea').first().type(value, { parseSpecialCharSequences: false })
+  cy.get('textArea').first().should('exist')
```

Comment:
I don't think this is necessary based on the line above.

Reply:
Oops. Forgot to set this PR to draft. I'm poking around here to figure out why the test got flaky recently.